src/ansys/grantami/recordlists/_connection.py

```diff
@@ -772,6 +772,9 @@ def search_for_audit_log_entries(
         """
         Fetch audit log entries, filtered by a search criterion.
 
+        If the ``criterion`` parameter does not specify a list identifier, then the results will be limited to the lists
```

Comment:
This is true but it's also true if the user does specify a criterion. I wonder if we move the comment about permissions to the headline since it applies in all cases?

Reply:
Isn't there a subtlety here between "lists that a user can see in the UI" and "lists that a user has permission to access". Generally, the reason you can't see a list in the UI is because you don't have permission to see it, except when a list is deleted, which is really the only time this distinction becomes significant. It's not really a comment about permissions, it's a sub-aspect of that.

I take your point though that the way it's currently written, it reads as if it's a comment about permissions.